atlp-rwanda/alpha-ec-be
# Ft update profile 187300152

### issue in referencing?

### Questions:

#### What does this PR do?

Update the user profile

#### Description of Task to be completed?
I want my profile to display my  
1. Personal Information 
   **name**, **email**,**gender**, **birthdate**, 
   **preferred language**, **preferred currency** 
  ,**where you live**, 

2. Billing Adress (check the right fields of the billing adress)

#### How should this be manually tested?


-run "npm run install"
-run "npm run migrate"
-run "npm run seed"
-run "npm dev"
- check the [swagger documentation ](http://localhost:3000/swagger/) on local

#### What are the relevant pivotal tracker/Trello stories?

- id of the project #187300152

#### Questions:
